src/pages/CamerasPage/CamerasPage.tsx

```diff
@@ -1,51 +1,43 @@
 import { CamerasFooter } from "./Footer/CamerasFooter";
 import styles from "./CamerasPage.module.scss";
 import { SecondaryCameras } from "./SecondaryCameras/SecondaryCameras";
-//import { useWebRTC } from "hooks/WebRTC/useWebRTC";
-import video1 from "./videos/videoAuran.mp4";
-import video2 from "./videos/videoIgnis.mp4";
-import video3 from "./videos/videoTurian.mp4";
-import { useState } from "react";
-import { CameraTitle } from "./TitleCamera/CameraTitle";
-
+import { useState, useCallback } from "react";
+import { CameraTitle } from "./CameraTitle/CameraTitle";
+import { CameraContainer } from "components/CameraContainer/CameraContainer";
+import { CameraData } from "./CameraData";
+import { LabeledCamera } from "./LabeledCamera/LabeledCamera";
 export const CamerasPage = () => {
```

Comment:
Add white line between imports and actual code, it is just a personal preference